terraform/modules/kubernetes/main.tf

```diff
@@ -57,12 +57,12 @@ resource "kubernetes_deployment" "webapp" {
           }
           resources {
             requests = {
-              cpu    = "100m"
-              memory = "256Mi"
+              cpu    = var.webapp_cpu_min
```

Comment:
As discussed, this could be a cluster-wide constant set in https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/terraform/aks/variables.tf#L101
1 for production, 0.1 for other clusters

Reply:
updated to try(var.cluster.cpu_min, 0.1)